### PR TITLE
Updated Flutter platform view demo for Android Java and Virtual Display

### PR DESCRIPTION
**Pull request by as1605**
**Notice:** Due to the upcoming holidays, PR reviews will be slower than usual. We hope that you have a wonderful holiday season. Happy 2022! – The Flutter Team

---

_Description of what this PR is changing or adding, and why:_
1. Fixed name of invalid view type for Virtual Display demo to match with the platform side view type.
2. Fixed error regarding `BinaryMessenger` argument while calling NativeViewFactory

_Issues fixed by this PR (if any):_
1. Earlier, Virtual Display demo used `'hybrid-view-type'` as viewType, but `"<platform-view-type>"` was registered on Android side, so it failed to call the view on run time.
2. Fixes https://github.com/flutter/website/issues/4825 for Java too, earlier only Kotlin snippet was fixed.

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by as1605** [comment]
I have signed the Contributor License Agreement (CLA). Please rescan/re-run

**Comment by domesticmouse** [comment]
@as1605 please update your PR to resolve conflicts, thanks!

File: src/development/platform-integration/platform-views.md [hunk 1]
```diff
@@ -317,13 +317,9 @@ import io.flutter.plugin.platform.PlatformViewFactory;
 import java.util.Map;
 
 class NativeViewFactory extends PlatformViewFactory {
-  @NonNull private final BinaryMessenger messenger;
-  @NonNull private final View containerView;
```
Comment (blasten) [inline src/development/platform-integration/platform-views.md:321]: I was trying to find why these were added in the first place, but couldn't see anything. 

This  matches the Kotlin example, and it's correct.

Just a comment, please remove the imports as well.

**Review by blasten — CHANGES_REQUESTED** [review]
remove the imports

**Review by blasten — APPROVED** [review]

**Comment by as1605** [comment]
@blasten Thanks for the review. I have removed the imports now

File: src/development/platform-integration/platform-views.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ import 'package:flutter/services.dart';
 ```dart
```
Comment (domesticmouse) [inline src/development/platform-integration/platform-views.md:137]: This is not the source of truth for this content. It is copied in from `lib/platform_views/native_view_example_2.dart` so you'll need to edit it there and then use the tooling to update this file.

@khanhnwin can supply more detail on how to run the snippet tooling.

Comment (khanhnwin) [reply src/development/platform-integration/platform-views.md:137]: Hi! Thanks for reminding me, I'll have to add instructions for the excerpts tooling to our readme. As @domesticmouse mentioned - once the Dart file with the code that is being copied in has changed, from the root of flutter/website you'll want to run the code excerpts script like so: `./tool/refresh-code-excerpts.sh` Let me know if you run into any issues.

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**